### ISSUE TITLE
Frustrating delete subscribers flow

### ISSUE THREAD

**Issue by ifpingram** [buttondown-email/roadmap#1422]
Following a search to find all my test users, using the gmail "+" syntax, e.g.

foo.bar+1@gmail.com
foo.bar+2@gmail.com
foo.bar+3@gmail.com
...

I then click the "View all results" at the bottom of the search results modal, in order to view the whole list on the page, as opposed to just the top results in the modal.

Then trying to delete them from this search results list, as soon as the delete button on the right is clicked, the page underneath refreshes with the default user list. I.e. it removes the search results from which I was running all the deletes from, thus necessitating a new search every time. 

(I have got around this by opening the delete function in a new tab FYI, but other's may not...)

Thanks!

**Comment by jmduke** [comment]
Agreed. I'm actually going to fold this into https://github.com/buttondown-email/roadmap/issues/1423 since they both will be fixed with the same improvement

**Comment by ifpingram** [comment]
This doesn't seem to have been fixed FYI. Same behaviour happening upon delete from search results.
